### ISSUE TITLE
Enable using tab key to navigate through checkboxes on first slide.

### ISSUE THREAD

**Issue by lisasy** [thepracticaldev/dev.to#6844]
This is an a11y-based bug based on P1's onboarding flow.

**Comment by vaidehijoshi** [comment]
@lisasy Do you have any more context you could provide on this? I was able to tab through the intro slide and check off the "code of conduct" and "terms and conditions" and then continued to the next slide.

Am I missing something? 🤔

**Comment by vaidehijoshi** [comment]
Closing this since @lisasy has confirmed that it no longer is an issue :)
